ts-bsp/sources/meta-quickphone/recipes-systemd-bbappend/systemd/systemd_219.bbappend

```diff
@@ -1,3 +0,0 @@
-SRC_URI_remove = " \
```

Comment:
bfc53e7bdc1d1a165f845648a60be91ecdb0ba55 was removing systemd udev patch because it was failing to apply on fido.
I did not have any issue with that on morty. A cleaner way would be to revert this patch and bump systemd version

Reply:
bfc53e7bdc1d1a165f845648a60be91ecdb0ba55 removed this patch to fix building issues. I did not have issues on morty. I would suggest to revert this commit before renaming the bbappend file.

Reply:
Okay I get it. The issue is that the commit message doesn't state anything about this bbappend being dropped, so it looks more like a mistake than a legitimate remove